### ISSUE TITLE
Auto-updating does not work (empty response)

### ISSUE THREAD

**Issue by SteffenL** [SteffenL/dise#4]
The updating service is non-operational and returns empty response.

**Comment by JjTheHusky** [comment]
If possible, probably best to make it so previous versions can alert users to a new update.

**Comment by SteffenL** [comment]
I agree but I do not think that is possible now. Older versions of DISE are really strict about the CA certificates they trust, and since I changed from StartCom to Comodo quite some time ago, DISE no longer trusts the new certificates. There is also no way to issue new StartCom certificates to make the old DISE work because the CA shut down the business. The new version of DISE uses SSL/TLS and CA trust provided by the operating system.
